src/partials/advantages.html

```diff
@@ -0,0 +1,9 @@
+<section class="section-advantages">
+  <h2 class="test">advantages</h2>
+  <h3>721</h3>
```

Comment:
cała ta sekcja to lista. Zastosuj <ul> i <li>

Reply:
Hej, nie chcę tego robić listą :)